### PR TITLE
Yakunina2

### PR DESCRIPTION
**Pull request by Natalia-Java-QA**
added TC_004 and TC_005 to AlexLeoEpicGroupTest

### CODE REVIEW

File: src/test/java/school/redrover/AlexLeoEpicGroupTest.java [hunk 1]
```diff
@@ -268,29 +266,26 @@ public void verifyTest() throws InterruptedException {
     }
 
 
+    @Test
+    public void testLoginPageIsOpenedTC_004() throws InterruptedException {
+        ChromeOptions chromeOptions = new ChromeOptions();
+        chromeOptions.addArguments("--remote-allow-origins=*", "--headless", "--window-size=1920,1080");
+        WebDriver driver = new ChromeDriver(chromeOptions);
```
Comment (evanmaiqa) [inline src/test/java/school/redrover/AlexLeoEpicGroupTest.java:273]: We are extending WebDriver from the BaseTest class.

Comment (Natalia-Java-QA) [reply src/test/java/school/redrover/AlexLeoEpicGroupTest.java:273]: yes, it's extended from BaseTest class in our group class (AlexLeoEpicGroupTest). My TCs are created honoring that extension.

Comment (evanmaiqa) [reply src/test/java/school/redrover/AlexLeoEpicGroupTest.java:273]: Is it not your test?
Ok. I see now. You first deleted and then reverted that test.

**Review by evanmaiqa — APPROVED** [review]

File: src/test/java/school/redrover/AlexLeoEpicGroupTest.java (hunk 1 above)
Comment (Natalia-Java-QA) [reply src/test/java/school/redrover/AlexLeoEpicGroupTest.java:273]: I'm sorry, I think I'm confused about what has happened and therefore I misled you.
I'm new to GitHub, so I'm sorry in advance.

I intended to add 2 my TCs: lines # 554-571 
I committed that change
Somehow I ended up with 2 commits: lines # 554-571 (which are my changes) and lines # 269-271 (I didn't change anything there, at least on purpose).

I don't want to have lines # 269-271 changed, as this is not my code. But I want to push my changes. What are my options here? Close this PR and create a new one, having there only my code (the last 2 TCs in the class)?

**Comment by Natalia-Java-QA** [comment]
some unexpected changes were committed.

**Pull request closed without merging**